backend/fms_core/models/experiment_type.py

```diff
@@ -34,4 +49,19 @@ def add_error(field: str, error: str):
     def save(self, *args, **kwargs):
         # Normalize and validate before saving, always!
         self.full_clean()
-        super().save(*args, **kwargs)  # Save the object
\ No newline at end of file
+        super().save(*args, **kwargs)  # Save the object
+
+    @property
+    def get_protocols_dict(self):
```

Comment:
This will likely be a method of workflows at some point ... I would comment it so we do not forget to relocate it someday ...